### DISCUSSION TITLE
Try to Sign-in With Different Account  and in terminal   [message: '_jose.EncryptJWT is not a constructor']

### DISCUSSION THREAD

**Discussion by arslansohail2** [nextauthjs/next-auth#4444] [Help]
### Environment

 [next-auth][error][SIGNIN_OAUTH_ERROR] 
https://next-auth.js.org/errors#signin_oauth_error _jose.EncryptJWT is not a constructor {
  error: {
    message: '_jose.EncryptJWT is not a constructor',
    stack: 'TypeError: _jose.EncryptJWT is not a constructor\n' +
      '    at Object.encode (/home/xoho/Documents/frontend/node_modules/next-auth/jwt/index.js:49:16)\n' +
      '    at async createState (/home/xoho/Documents/frontend/node_modules/next-auth/core/lib/oauth/state-handler.js:29:24)\n' +
      '    at async getAuthorizationUrl (/home/xoho/Documents/frontend/node_modules/next-auth/core/lib/oauth/authorization-url.js:70:19)\n' +
      '    at async Object.signin (/home/xoho/Documents/frontend/node_modules/next-auth/core/routes/signin.js:37:24)\n' +
      '    at async NextAuthHandler (/home/xoho/Documents/frontend/node_modules/next-auth/core/index.js:191:26)\n' +
      '    at async NextAuthNextHandler (/home/xoho/Documents/frontend/node_modules/next-auth/next/index.js:21:19)\n' +
      '    at async /home/xoho/Documents/frontend/node_modules/next-auth/next/index.js:57:32\n' +
      '    at async Object.apiResolver (/home/xoho/Documents/frontend/node_modules/next/dist/server/api-utils.js:101:9)\n' +
      '    at async DevServer.runApi (/home/xoho/Documents/frontend/node_modules/next/dist/server/next-server.js:320:9)\n' +
      '    at async Object.fn (/home/xoho/Documents/frontend/node_modules/next/dist/server/base-server.js:486:37)',
    name: 'TypeError'
  },
  provider: {
    id: 'github',
    name: 'GitHub',
    type: 'oauth',
    authorization: {
      url: 'https://github.com/login/oauth/authorize',
      params: [Object]
    },
    token: { url: 'https://github.com/login/oauth/access_token', params: {} },
    userinfo: {
      url: 'https://api.github.com/user',
      request: [AsyncFunction: request]
    },
    profile: [Function: profile],
    idToken: false,
    checks: [ 'state' ],
    clientId: ' ',
    clientSecret: ' ',
    signinUrl: 'http://localhost:3000/api/auth/signin/github',
    callbackUrl: 'http://localhost:3000/api/auth/callback/github'
  },
  message: '_jose.EncryptJWT is not a constructor'

![Screenshot from 2022-04-22 11-28-20](https://user-images.githubusercontent.com/100696713/164616393-219a9eb6-4e83-4e01-a48c-bd05198da516.png)
 

### Reproduction URL

 https://github.com/nextauthjs/next-auth-example

### Describe the issue

I use the .env file and verify 100 times my Client id or Client secrets in other project i Signin it become signed but in my main project i click sign in it faces error TRY TO SIGN IN WITH DIFFERENT ACCOUNT and in terminal their is this kind of error 

[next-auth][error][SIGNIN_OAUTH_ERROR] 
https://next-auth.js.org/errors#signin_oauth_error _jose.EncryptJWT is not a constructor {
  error: {
    message: '_jose.EncryptJWT is not a constructor',
    stack: 'TypeError: _jose.EncryptJWT is not a constructor\n' +
      '    at Object.encode (/home/xoho/Documents/frontend/node_modules/next-auth/jwt/index.js:49:16)\n' +
      '    at async createState (/home/xoho/Documents/frontend/node_modules/next-auth/core/lib/oauth/state-handler.js:29:24)\n' +
      '    at async getAuthorizationUrl (/home/xoho/Documents/frontend/node_modules/next-auth/core/lib/oauth/authorization-url.js:70:19)\n' +
      '    at async Object.signin (/home/xoho/Documents/frontend/node_modules/next-auth/core/routes/signin.js:37:24)\n' +
      '    at async NextAuthHandler (/home/xoho/Documents/frontend/node_modules/next-auth/core/index.js:191:26)\n' +
      '    at async NextAuthNextHandler (/home/xoho/Documents/frontend/node_modules/next-auth/next/index.js:21:19)\n' +
      '    at async /home/xoho/Documents/frontend/node_modules/next-auth/next/index.js:57:32\n' +
      '    at async Object.apiResolver (/home/xoho/Documents/frontend/node_modules/next/dist/server/api-utils.js:101:9)\n' +
      '    at async DevServer.runApi (/home/xoho/Documents/frontend/node_modules/next/dist/server/next-server.js:320:9)\n' +
      '    at async Object.fn (/home/xoho/Documents/frontend/node_modules/next/dist/server/base-server.js:486:37)',
    name: 'TypeError'
  },
  provider: {
    id: 'github',
    name: 'GitHub',
    type: 'oauth',
    authorization: {
      url: 'https://github.com/login/oauth/authorize',
      params: [Object]
    },
    token: { url: 'https://github.com/login/oauth/access_token', params: {} },
    userinfo: {
      url: 'https://api.github.com/user',
      request: [AsyncFunction: request]
    },
    profile: [Function: profile],
    idToken: false,
    checks: [ 'state' ],
    clientId: ' ',
    clientSecret: ' ',
    signinUrl: 'http://localhost:3000/api/auth/signin/github',
    callbackUrl: 'http://localhost:3000/api/auth/callback/github'
  },
  message: '_jose.EncryptJWT is not a constructor'

please help

### How to reproduce

how can i solve this error

### Expected behavior

 cc

**Comment by Lyazarus** [comment]
Hello,
I have this issue too but with nuxt 3

```
[nuxt] [request error] [unhandled] [500] _jose.EncryptJWT is not a constructor
  at Object.encode (/webapp/server/node_modules/next-auth/jwt/index.js:49:16)
  at async Object.callback (/webapp/server/node_modules/next-auth/core/routes/callback.js:432:22)
  at async AuthHandler (/webapp/server/node_modules/next-auth/core/index.js:302:28)
  at async /webapp/server/chunks/nitro/node-server.mjs:1753:24
  at async Object.handler (/webapp/server/chunks/nitro/node-server.mjs:1015:19)
  at async Server.toNodeHandle (/webapp/server/chunks/nitro/node-server.mjs:1069:7)

```
Do you find a solution on your side @arslansohail2 ?
